src/plugins/backend/README.md

```diff
@@ -5,7 +5,7 @@
 - infos/needs =
 - infos/recommends =
 - infos/placements = backend
-- infos/status = nodep
+- infos/status = unittest nodep configurable nodoc
```

Comment:
Did you verify all these changes to be correct?

Reply:
On behalf of what are those changes made? I don‘t really understand the `infos-status-update` script 😵‍💫